### DISCUSSION TITLE
How about online office suite integration (Collabora, Onlyoffice) ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#15573] [Wishlist]
Hi,

I've just give a try with Collabora office built-in server available in Nextcloud and it's really nice to edit documents without having to download et re-upload again when you need to make some modifications.
https://apps.nextcloud.com/apps/richdocumentscode

The collabora built-in server is interesting for small business because it doesn't require to build your own collabora office server.
https://github.com/CollaboraOnline

On the same subject, I also found an interesting symfony bundle + a WOPI connector.
I didn't know about WOPI (Web Application Open Platform Interface Protocol) before searching for office online editors.
https://github.com/Champs-Libres/wopi-lib
https://github.com/Champs-Libres/wopi-bundle
https://www.youtube.com/watch?v=6FA7kdGi2rs
https://people.gnome.org/~michael/data/2020-02-01-web-collaboration.pdf

Using WOPI, it is also possible to connect to Microsoft Office Online Server.
https://apps.nextcloud.com/apps/officeonline

Regards,

**Comment by DeMezzo** [comment]
That would be great and very helpful in this system
